### DISCUSSION TITLE
open shell and multiple desktop on windows 11

### DISCUSSION THREAD

**Discussion by axolotus** [Open-Shell/Open-Shell-Menu#1697] [General]
Hi,
I've installed Open Shell 4.4.191 on Windows 11, but I'm unable to use the "View Task" icon to launch multiple desktops. Can I use Open Shell with multiple desktops, or is there a bug?
Thank you for your help.

**Comment by ge0rdi** [comment]
> I'm unable to use the "View Task" icon to launch multiple desktops

What exactly doesn't work for you?
Can you provide aby steps for what you are trying to do?
Eventually some screenshots for clarification?

> Can I use Open Shell with multiple desktops, or is there a bug?

I'm able to click on `Task view` button, add virtual desktops and switch between them on Windows 11 22H2 with `Open-Shell` 4,4,191 without aby problem. So I'd say it is supposed to work.

What Windows 11 version do you have?

**Comment by axolotus** [comment]
Hi,
Thank you for your prompt response.
I'm using Windows 11 22H2, and I have Open Shell 4.4.191 installed.

When Open Shell is not installed, I have the correct icon for the task view, and I can open multiple desktops. However, when I install Open Shell, the icon looks like the Windows 10 icon, and I am unable to launch multiple desktops.
I've attached a screenshot of the icon I'm referring to.
![icon_open_shell](https://github.com/Open-Shell/Open-Shell-Menu/assets/5745958/0b22960e-2aa5-44e8-8cb9-acd7bf62e3db)

**Reply by ge0rdi** [reply]
That's pretty weird. `Open-Shell` is not supposed to change taskbar icons.

Do you use other customization software, like `ExplorerPatcher`?
Does the issue go away when you _exit_ `Open-Shell` (right-click on start button and Exit)?

**Comment by axolotus** [comment]
Hi,

Excuse me. I made a mistake. I thought it was related to OpenShell. Yes, I installed ExplorerPatcher. OpenShell works perfectly.
Thank you for your valuable assistance. Everything is back to normal.
